pki.spec

```diff
@@ -452,6 +465,22 @@ Requires:         python2-pki = %{version}-%{release}
 Requires(post):   python2-pki = %{version}-%{release}
 %endif  # with_python3_default
 
+# Ensure we end up with a useful installation
+Conflicts:        pki-symkey < %{version}-%{release}
+Conflicts:        pki-server < %{version}-%{release}
+Conflicts:        pki-ca < %{version}-%{release}
+Conflicts:        pki-kra < %{version}-%{release}
+Conflicts:        pki-ocsp < %{version}-%{release}
+Conflicts:        pki-tks < %{version}-%{release}
+Conflicts:        pki-tps < %{version}-%{release}
+Conflicts:        pki-javadoc < %{version}-%{release}
+Conflicts:        pki-console < %{version}-%{release}
+Conflicts:        dogtag-pki < %{version}-%{release}
+Conflicts:        pki-base-java < %{version}-%{release}
+Conflicts:        pki-tools < %{version}-%{release}
+Conflicts:        pki-server-theme < %{version}-%{release}
+Conflicts:        pki-console-theme < %{version}-%{release}
```

Comment:
pki-base should only need to conflict with other top-level packages in the graph:
* pki-symkey
* pki-javadoc
* pki-server-theme
* pki-console-theme